crates/libtiny_tui/src/tui.rs

```diff
@@ -377,6 +384,38 @@ impl TUI {
             ret
         };
 
+        // Get tab configs for the type of tab being created
+        let TabConfig { ignore, notifier } = if let Some(config) = self.load_config() {
+            match &src {
+                MsgSource::Serv { serv } => config.server_tab_configs(serv),
+                MsgSource::Chan { serv, chan } => {
+                    let tab_configs = config.chan_tab_configs(serv, chan);
+                    // Prioritize config file but fallback to current server tab settings (when Defaults are not set)
+                    let server_tab_config = self
+                        .tabs
+                        .iter()
+                        .find_map(|tab| {
+                            if tab.src.serv_name() == serv {
+                                Some(TabConfig {
+                                    ignore: Some(!tab.widget.is_showing_status()),
+                                    notifier: Some(tab.notifier),
+                                })
+                            } else {
+                                None
+                            }
+                        })
```

Comment:
```suggestion
                    let server_tab_config = self
                        .tabs
                        .iter()
                        .find(|tab| tab.src.serv_name() == serv)
                        .map(|tab| TabConfig {
                            ignore: Some(!tab.widget.is_showing_status()),
                            notifier: Some(tab.notifier),
                        })
```
 
Same for other uses of `find_map`.